tests/test_app_basic.py

```
@@ -0,0 +1,164 @@
+"""
+app.pyの基本的な統合テスト（リファクタリング版）
+"""
+import pytest
+from flask import Flask
```

Ruff (0.12.2)

`flask.Flask` imported but unused

Remove unused import: `flask.Flask`

(F401)